web/client/components/map/leaflet/__tests__/Map-test.jsx

```diff
@@ -465,6 +468,67 @@ describe('LeafletMap', () => {
         expect(getPixelFromCoordinates).toExist();
         expect(getCoordinatesFromPixel).toExist();
     });
+    it('test ZOOM_TO_EXTENT_HOOK', () => {
+        mapUtils.registerHook(mapUtils.ZOOM_TO_EXTENT_HOOK, undefined);
+
+        const testHandlers = {
+            onMapViewChanges: () => {}
+        };
+        // fix size
+        document.querySelector('#container').setAttribute('style', "width: 200px; height: 200px");
+        const spy = expect.spyOn(testHandlers, 'onMapViewChanges');
+        const map = ReactDOM.render(<LeafletMap
+            id="mymap"
+            center={{ y: 0, x: 0 }}
+            zoom={11}
+            registerHooks
```

Comment:
register hook is true by default in every maptype, so this should not be needed. is that right?